odoo/cli/db.py

```diff
@@ -39,6 +48,55 @@ def run(self, cmdargs):
         parser.set_defaults(func=lambda _: exit(parser.format_help()))
 
         subs = parser.add_subparsers()
+
+        # INIT ----------------------------------
+
+        init = subs.add_parser(
+            "init",
+            help="Create and initialize a database",
+            description="Create an empty database and install the minimum required modules",
+            formatter_class=RawTextHelpFormatter,
+        )
+        init.set_defaults(func=self.init)
+        init.add_argument(
+            'database', nargs='?', default='default',
```

Comment:
Could you confirm you think the `default` is useful?

Reply:
Removed.